### PR TITLE
CVE release 2.28.4 + fix GLIBC go plugin build issue

### PR DESCRIPTION
**Pull request by PettitWesley**
*Issue #, if available:*

#463

*Description of changes:*

I tested this change with `make release` and then verified that each go plugin will load and run.

I then ran `make windows-plugins` to make sure it succeeds. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Dockerfile.plugins [hunk 1]
```diff
@@ -54,26 +51,4 @@ WORKDIR /cloudwatch
...
-
-# Copy binaries
-RUN cp /kinesis-streams/bin/*.so /kinesis-firehose/bin/*.so /cloudwatch/bin/*.so /plugins/linux
-RUN cp /kinesis-streams/bin/*.dll /kinesis-firehose/bin/*.dll /cloudwatch/bin/*.dll /plugins/windows
-
-# Copy licenses
-RUN cp /kinesis-streams/THIRD-PARTY /kinesis-streams/LICENSE /plugins/linux/licenses/kinesis
-RUN cp /kinesis-firehose/THIRD-PARTY /kinesis-firehose/LICENSE /plugins/linux/licenses/firehose
-RUN cp /cloudwatch/THIRD-PARTY /cloudwatch/LICENSE /plugins/linux/licenses/cloudwatch
-RUN cp /kinesis-streams/THIRD-PARTY /kinesis-streams/LICENSE /plugins/windows/licenses/kinesis
```
Comment (PettitWesley) [inline Dockerfile.plugins:74]: I simply reverted this file back to the version we had in 2.28.3

@rawahars added these lines to copy stuff  out of the image, which is used in the windows pipeline.

We can consider adding that back in for linux when he is available to chat on it. 

The Dockerfile copies the License and third party files from the plugins into the image already. This does not impact our compliance.

**Review by ShelbyZ — APPROVED** [review]

**Review by matthewfala — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+### 2.28.4
+This release includes:
+* Fluent Bit [1.9.9](https://fluentbit.io/announcements/v1.9.9/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.0
```
[the remark this answers is not in the document]
Comment (PettitWesley) [reply CHANGELOG.md]: fixed

File: Dockerfile.plugins-windows [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+WORKDIR /cloudwatch
+RUN if [ -n "$CLOUDWATCH_PLUGIN_BRANCH" ];then git fetch --all && git checkout $CLOUDWATCH_PLUGIN_BRANCH && git remote -v;fi
+RUN if [ -z "$CLOUDWATCH_PLUGIN_BRANCH" ];then git fetch --all --tags && git checkout tags/$CLOUDWATCH_PLUGIN_TAG -b $CLOUDWATCH_PLUGIN_TAG && git describe --tags;fi
+RUN go mod download
+RUN make windows-release
+
+# Add all the built artifacts to an output folder and create an archive
+# We can copy this to the host for saving the artifacts
+
+RUN mkdir -p /plugins/linux/licenses/kinesis /plugins/linux/licenses/firehose /plugins/linux/licenses/cloudwatch
```
[the remark this answers is not in the document]
Comment (PettitWesley) [reply Dockerfile.plugins-windows:62]: oh thanks forgot about this

[the remark this answers is not in the document]
Comment (PettitWesley) [reply Dockerfile.plugins-windows]: fixed

**Pull request merged**